config/default.json

```diff
@@ -49,6 +49,11 @@
             "yarpName": "${this.robotYarpPortPrefix}/battery/data:o",
             "localName": "${this.localYarpPortPrefix}/battery/data:i",
             "portType": "bottle"
+        },
+        "walkingController.logger": {
+            "yarpName": "/walking-coordinator/logger/data:o",
+            "localName": "${this.localYarpPortPrefix}/walking-coordinator-logger/data:i",
+            "portType": "bottle"
         }
```

Comment:
Not a comment blocking this review, but it would be cool if adding a new port that is publishing a `BipedalLocomotion.YarpUtilities.VectorsCollection` data type could be added just by adding a new json element like:
~~~
        "otherModule.logger": {
            "yarpName": "/otherModule/logger/data:o",
            "localName": "${this.localYarpPortPrefix}/otherModule-logger/data:i",
            "portType": "BipedalLocomotion.YarpUtilities.VectorsCollection"
        }
~~~

This would permit to add in the telemetry logger data coming from multiple modules, without any modification to the `.js` file. Anyhow, for sure we can merge the PR as it is!

Reply:
Indeed. This could be the starting point for the already opened issue https://github.com/ami-iit/yarp-openmct/issues/45.

Reply:
Addressed in #45